profile_xf05id1/startup/15-optics.py

```diff
@@ -2,9 +2,24 @@
 from ophyd import Device
 from ophyd import Component as Cpt
 
+class SRXSlitsWB(Device):
+    #THIS IS BROKEN AT EPICS :LEVEL
```

Comment:
Thanks for bringing it up, @danielballan. I don't recall the exact issue @tacaswell and I were discussing, but it would be great if from now on if we could start recording such issues. It would benefit not only us but the beamlines and their control engineers as well.